src/plugins/transform/scss.ts

```diff
@@ -44,10 +44,10 @@ export function transformScssPlugin({ options }: ModuleContext): VitePlugin {
 }
 
 function createScssTransform(fileExtension: string, sassVariables?: string | boolean): (code: string) => string {
-  const sassImportCode = ['@import \'quasar/src/css/variables.sass\'', '']
+  const sassImportCode = ['@use \'quasar/src/css/variables.sass\' as *', '']
 
   if (typeof sassVariables === 'string') {
-    sassImportCode.unshift(`@import '${sassVariables}'`)
+    sassImportCode.unshift(`@use '${sassVariables}'`)
```

Comment:
```suggestion
    sassImportCode.unshift(`@use '${sassVariables}' as *`)
```

I'm inclined to change this to `as *` as well to not break the config of users as it otherwise requires users to prefix their SASS variables in components. However, this does mean users can only supply just the path and it will always be imported without prefix

users can keep doing this:

```ts
sassVariables: '@/assets/scss/_variables.scss',
```

but cannot change it themselves between:

```ts
		sassVariables: '@/assets/scss/_variables.scss as prefix',
OR
		sassVariables: '@/assets/scss/_variables.scss',
```

Reply:
We can also accept a object for `sassVariables` option. With default behavior using `as  *`. (`prefix` being empty string or '*')

```js
{
  sassVariables: {
    file: './assets/scss/_variables.scss',
    prefix: '*'
  }
}
```
should work same as
```js
{
  sassVariables: './assets/scss/_variables.scss'
}
```

Reply:
Nevermind,  I was not aware that with `@use`, variables cannot be simply overriden.